### ISSUE TITLE
Rebar3 plugin?

### ISSUE THREAD

**Issue by drozzy** [erldocs/erldocs#54]
Is it possible to use this as a rebar3 plugin?

I.e. I currently run edoc on my project as follows:

`rebar3 edoc`

Would be nice to have:

`rebar3 erldoc`

Thanks!

**Comment by fenollp** [comment]
Yes it would! Do you feel like you can contribute it?

**Comment by fenollp** [comment]
Adding a rebar.config (for rebar3) is also something I lack time to do, but should be very easy!

**Comment by drozzy** [comment]
Sorry, I've no idea how to go about this.

**Comment by fenollp** [comment]
This should help https://github.com/erldocs/erldocs/pull/58
(it's a first step)

**Comment by drozzy** [comment]
What would be the proper way to call this from erlang?

Something like this?

    erladocs:main(["."]).

**Comment by fenollp** [comment]
Yes. BTW my rebar.config isn't correctly compiling erlydtl templates yet, so this might not work yet.

**Comment by fenollp** [comment]
Rebar config fixed now. Package up on hex.pm: https://hex.pm/packages/erldocs
